packages/react/src/index.js

```diff
@@ -1,7 +1,11 @@
 // @flow
 import pkg from '../package.json'
 export type { SerializedStyles } from '@emotion/utils'
-export { withEmotionCache, CacheProvider } from './context'
+export {
+  withEmotionCache,
+  CacheProvider,
+  useEmotionCacheContext
+} from './context'
```

Comment:
The naming here feels a bit strange:

- Why do we have `CacheProvider ` not mentioning "emotion" but `withEmotionCache` does?
- Why do we have `withEmotionCache` not mentioning "context" but `useEmotionCacheContext` does?

Reply:
Agreed 😞 I'm stupid for putting `'Context'`,
However `withEmotionCache` and `CacheProvider` were named this way already. It's in the diff only because prettier rearranged the statement because it was becoming too long for a single line.  
I assume it is out of the question to introduce breaking change for fixing name inconsistency.  
That in mind what would you see as a better name for the hook?  
- `useCache` or
- `useEmotionCache`  
I personally prefer `useEmotionCache`, it is more consistent with `withEmotionCache` with which it shares the same goal of providing access to the cache.

Reply:
I have rebased to export `useEmotionCache` instead of `useEmotionCacheContext`

Reply:
Ok, in any case, I'm not a maintainer of this GitHub repository